presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/glue/GlueHiveMetastoreConfig.java

```diff
@@ -163,4 +166,33 @@ public GlueHiveMetastoreConfig setAwsCredentialsProvider(String awsCredentialsPr
         this.awsCredentialsProvider = Optional.ofNullable(awsCredentialsProvider);
         return this;
     }
+
+    @Min(1)
+    @Max(10)
```

Comment:
Why 10 and not eg 100?

Reply:
Glue segment can range from 1 to 10. https://docs.aws.amazon.com/glue/latest/webapi/API_Segment.html